### PR TITLE
Copied Why perflow.js and .css to fix the previous branch issues

### PR DESCRIPTION
**Pull request by MiladYaqo**
**LayoutImprove#3**
Had problems with Fe-Why-Perflow-Page-create so pushed a new branch.
The changes:
-Added a picture as a background for the text.
-Added an image to the source.
-Created separate Css for Why Perflow page.
-Removed some CSS code from App.css, the removed CSS were added into Whyperflow.Css - a separate file.

### CODE REVIEW

File: frontend/src/components/WhyPerflow/WhyPerflow.css [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        color:black;
+    }
+
+    #card1column, #card2column{
+
+    }
+   
+    #title1, #title2{
+      font-size: 80px;
+      background-image: url("/perflowShorterlowsat.png");
```
Comment (MelanyCristy) [inline frontend/src/components/WhyPerflow/WhyPerflow.css]: Nice job here! It’s really nice to have all the code in their corresponding place! 👍

File: frontend/src/components/WhyPerflow/WhyPerflow.js [hunk 2]
```diff
@@ -1,20 +1,29 @@
 import React from 'react';
+import './WhyPerflow.css';
+
 
 class WhyPerflow extends React.Component {
     render() {
+
```
Comment (MelanyCristy) [inline frontend/src/components/WhyPerflow/WhyPerflow.js:7]: Blank space, please remove

File: frontend/src/components/WhyPerflow/WhyPerflow.js [hunk 3]
```diff
@@ -1,20 +1,29 @@
 import React from 'react';
+import './WhyPerflow.css';
+
```
Comment (MelanyCristy) [inline frontend/src/components/WhyPerflow/WhyPerflow.js:3]: Blank space, please remove

File: frontend/src/components/WhyPerflow/WhyPerflow.js [hunk 4]
```diff
@@ -1,20 +1,29 @@
...
         return (
-            <React.Fragment className="loginPage">
+            <React.Fragment>
+                <div className= "container1">
                 <div className="whyChoosePerflowSection">
-                    <div className="TitleDiv text-center">
-                        <h1 className="Title text-white">Why Choose Perflow?</h1>
+                    <div className="TitleDiv text-center" >
+                        <h1 className="Title" id="title1">Why Choose Perflow?</h1>
+
```
Comment (MelanyCristy) [inline frontend/src/components/WhyPerflow/WhyPerflow.js:14]: Blank space, please remove.

File: frontend/src/components/WhyPerflow/WhyPerflow.js [hunk 5]
```diff
@@ -27,27 +36,50 @@ class WhyPerflow extends React.Component {
...
-                            <div className="col-sm-6">
-                                <div>
-                                    <h1 className="text-center">"Since my team started using PerFlow, we've communicated better - both with each
-                                other, as well as with our clients. We're able to give clients an updated, clear picture of how their project is going." </h1>
-                                    <p className="text-right">--Kjell Gustavson, Project Manager and PerFlow user</p>
+
+                            <div className="col-lg-4 mt-lg-4 mr-lg-auto mb-lg-5 ml-lg-1
+                                            col-md-6 ml-md-auto mt-md-3 mr-md-auto mb-md-3
+                                            col-sm-6 mr-sm-6" id="card2column">
+
```
Comment (MelanyCristy) [inline frontend/src/components/WhyPerflow/WhyPerflow.js:64]: Blank space, please remove

**Review by MelanyCristy — APPROVED** [review]
Looks nice to me. Just some blank spaces to remove

**Review by jshortz — APPROVED** [review]
Loaded and reviewed in web and reviewed code

**Pull request merged**